docs/docker-compose-ca.yml.example

```diff
@@ -0,0 +1,18 @@
+# Example docker-compose file for the SSH CA.
+version: '3.7'
+services:
+  kbsshca:
+    image: ca:latest
+    container_name: kbsshca
+    #command: ["./docker/entrypoint-server.sh"]
+    #command: ["./docker/entrypoint-generate.sh"]
+    command: ["sh", "-c", "sleep 1000"]
```

Comment:
I think your idea here is to have someone manually comment out the correct line to use as the entrypoint, right? If so, fine by me but probably just add a comment to this file here explaining this.

Reply:
commented